.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,29 @@
+name: e2e
+on:
+  pull_request:
+    paths-ignore:
+    - '**/*.md'
```

Comment:
Just a quick note: we'll run into issues with statically filtering out certain PRs if/when we setup the required checks through branch protection in this repository, but this is fine for now.

Reply:
Could you help me to understand what problem this line causes to that end?

Reply:
Basically the static filtering results in checks reporting an "unknown" state which is incompatible with existing branch protection rules that require successful states to gate merge criteria. You'd need to move towards dynamically filtering out certain PRs to workaround this scenario. I haven't found a really clean way of tackling this problem so I've mainly landed on removing the static filtering entirely, and thus wasting unnecessary CI runs on markdown-only PRs.

Reply:
Got it, when we end up adding the branch-protection rules like you're mentioning lets readdress this.